packages/playwright-core/src/utils/traceUtils.ts

```diff
@@ -120,6 +120,10 @@ export async function saveTraceFile(fileName: string, traceEvents: ActionTraceEv
   });
 }
 
+export function maskString(str: string) {
+  return '*'.repeat(str.length);
```

Comment:
We should not leak the secret's length as well:

```suggestion
  return '****';
```